### PR TITLE
[AC-659] Add Contact Us Screen.

### PR DESCRIPTION
**Pull request by ribhavsharma**
## Description of what I changed

Added a Contact Us Screen that sends an email to the OpenMRS HelpDesk.  
- [x] Added it in the overflow menu.
- [x] Added it in the settings fragment.

## Screenshot
Screenshot_20200117-220611

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-659

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import android.content.Intent;
+import android.view.View;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+import com.google.android.material.button.MaterialButton;
+import com.google.android.material.textfield.TextInputEditText;
+import org.openmrs.mobile.R;
+
+public class ContactUsActivity extends AppCompatActivity {
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: use ACBaseActivity instead to implement dark mode and language change features...

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java [hunk 2]
```diff
@@ -80,6 +81,7 @@ public void onResume() {
         super.onResume();
         mPresenter.updateConceptsInDBTextView();
         LocalBroadcastManager.getInstance(this.getActivity()).registerReceiver(bReceiver, new IntentFilter(ApplicationConstants.BroadcastActions.CONCEPT_DOWNLOAD_BROADCAST_INTENT_ID));
+        contactUs();
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsFragment.java]: You can write this in setUpConceptsView() functions instead

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsContract.java [hunk 3]
```diff
@@ -34,6 +34,8 @@
         void rateUs();
 
         void setDarkMode();
+
+        void contactUs();
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsContract.java]: There is no need to add it if you are not calling it from SettingsPresenter.java

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_contact_us);
+
+        send = findViewById(R.id.send);
+        name = findViewById(R.id.name);
+        subject = findViewById(R.id.subject);
+        message = findViewById(R.id.message);
+
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: Bind them in different function **initViews()** to maintain architecture of the previous code

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsPresenter.java [hunk 5]
```diff
@@ -14,7 +14,12 @@
 
 package org.openmrs.mobile.activities.settings;
 
+import android.content.Intent;
+import android.view.View;
+import android.widget.LinearLayout;
+import org.openmrs.mobile.R;
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsPresenter.java]: Seems to be unused imports

**Comment by ribhavsharma** [comment]
thanks for the suggestions @rishabh-997, I'll be committing when I get home :)

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsPresenter.java (hunk 5 above)
Comment (ribhavsharma) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsPresenter.java]: which one?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java (hunk 4 above)
Comment (ribhavsharma) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java]: done

**Comment by saadkhaleeq610** [comment]
Resolve conflicts

**Pull request closed without merging**